### PR TITLE
Indentation fixes

### PR DESCRIPTION
**Pull request by hhomar**
Hi

Just a couple of small indentation fixes found while testing 4.38.0.

### CODE REVIEW

**Comment by james-d-elliott** [comment]
Please include examples of what this is intending on fixing as it's not clear from the changes and tests I've done.

**Comment by hhomar** [comment]
# wrapYAML indentation

With the following access_control set in values.yml:

```yaml
  access_control:
    secret:
      enabled: true
      key: 'configuration.acl.yaml'
      existingSecret: ''
    default_policy: 'deny'
    networks: []
    rules:
      - domain: '*.example.com'
        policy: 'one_factor'
```

This was generating a secret with an invalid access control yaml file (indentation of access_control:):

```yaml
---
        access_control:
  default_policy: 'deny'
  rules:
    - policy: one_factor
      domain:
        - '*.example.com'
    ...
```

# oidc jwks  key indentation

With the following identity_providers oidc jwks key (other values left out for brevity):

```yaml
identity_providers:
  oidc:
    enabled: true
    jwks:
      - key:
          path: "/secrets/authelia-jwks/private.pem"
```

This would generate a secret template with an invalid indentation (mindent 8 instead of mindent 12):

```yaml
jwks:
  key: {{ secret "/secrets/authelia-jwks/private.pem" | mindent 8 "|" | msquote }}
```

Causing the following error:

Configuration: failed to load configuration from file
path(/configuration.yaml) source: yaml: line 158: could not find
expected ':'

**Comment by james-d-elliott** [comment]
Ah yep, that makes sense. I'll double check it.

File: charts/authelia/templates/configMap.yaml [hunk 1]
```diff
@@ -397,7 +397,7 @@ data:
         {{- end }}
         jwks:
         {{- range $key := .Values.configMap.identity_providers.oidc.jwks }}
-        {{- tpl ($.Files.Get "files/configuration.oidc.jwk.yaml") (merge (dict "Key" $key "Indent" 4) $) | trim | nindent 10 }}
+        {{- tpl ($.Files.Get "files/configuration.oidc.jwk.yaml") (merge (dict "Key" $key "Indent" 8) $) | trim | nindent 10 }}
```
Comment (james-d-elliott) [inline charts/authelia/templates/configMap.yaml:400]:
```suggestion
        {{- tpl ($.Files.Get "files/configuration.oidc.jwk.yaml") (merge (dict "Key" $key "Indent" 6) $) | trim | nindent 10 }}
```

**Review by james-d-elliott — CHANGES_REQUESTED** [review]
Just this one minor change which should fix it but keep the indent consistent.

**Pull request merged**